packages/playground/website/playwright/e2e/query-api.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { test, expect } from '../playground-fixtures';
+
+// We can't import the WordPress versions directly from the remote package
+// because of ESModules vs CommonJS incompatibilities. Let's just import the
+// JSON file directly. @ts-ignore
+// eslint-disable-next-line @nx/enforce-module-boundaries
+import * as MinifiedWordPressVersions from '../../../wordpress-builds/src/wordpress/wp-versions.json';
+
+const LatestSupportedWordPressVersion = Object.keys(
+	MinifiedWordPressVersions
+).filter((x) => !['nightly', 'beta'].includes(x))[0];
+
+test('should load PHP 8.0 by default', async ({ page, wordpressPage }) => {
+	// Navigate to the page
+	await page.goto('./?url=/phpinfo.php');
+
+	// Find the h1 element and check its content
+	const h1 = wordpressPage.locator('h1.p').first();
+	await expect(h1).toContainText('PHP Version 8.0');
+});
+
+test('should load WordPress latest by default', async ({
+	page,
+	wordpressPage,
+}) => {
+	await page.goto('./?url=/wp-admin/');
+
+	const expectedBodyClass =
+		'branch-' + LatestSupportedWordPressVersion.replace('.', '-');
+	const body = wordpressPage.locator(`body.${expectedBodyClass}`);
```

Comment:
Classes can also be expected with Playwright:
```suggestion
	const body = wordpressPage.locator('body');
	await expect(body).toHaveClass('branch-' + LatestSupportedWordPressVersion.replace('.', '-'));
```

Reply:
This doesn't work for my usecase. 
When the iframe loads the `body` is empty at first, but Playwright takes that body and fails the test because the class isn't there. If I add the class to the locator it waits for the class which is added shortly after the iframe loads. 
But I don't know much about Playwright and there's a chance I'm just doing something wrong.

Reply:
A class added asynchronously is a typical scenario supported by the `toHaveClass` assertion, as (like other locator assertions) it's polling the attribute until it matches the expected value (or timeout). Would you be able to paste the exact error being thrown here? 🤔

Reply:
I tried it again, it just timesout.  I can see the page loaded in the preview.